charts/nexus-repository-manager/README.md

```diff
@@ -136,8 +136,9 @@ The following table lists the configurable parameters of the Nexus chart and the
 | `service.labels`                            | Service labels                   | `nil`                                      |
 | `service.annotations`                       | Service annotations              | `nil`                                      |
 | `service.type`                              | Service Type                     | `ClusterIP`                                |
-| `route.enabled`         | Set to true to create route for additional service | `false` |
-| `route.name`            | Name of route                                      | `docker` |
+| `nexusUIRoute.enabled`         | Set to true to create route for Nexus UI (for OpenShift) | `false` |
```

Comment:
I'm not sure about calling in "nexusUIRoute", since this is the http interface for _everything_ nexus: the UI, getting and storing artifacts.  I might want to just call it "nexusRoute".  I'm not sure the old "nexusProxyRoute" name added much description, so i'm fine with changing it.

Reply:
Picked nexusUIRoute for port 8080, to differentiate between routes for docker services. **nexusRoute** name would be more suitable, if we only have one Route.  I still couldn't come up with a name for docker routes, as we may need to create minimum 2, 1 for hosted repo and 1 for group / proxy. Any suggestion would be great.